optuna_dashboard/_preferential_history.py

```diff
@@ -23,13 +23,24 @@
         {
             "mode": FeedbackMode,
             "id": str,
-            "preference_id": str,
             "timestamp": str,
             "candidates": list[int],
             "clicked": int,
+            "preferences": list[tuple[int, int]],
         },
     )
     History = ChooseWorstHistory
+    SerializedHistory = TypedDict(
+        "SerializedHistory",
+        {
+            "history": History,
+            "is_removed": bool,
+        },
+    )
+
+
+class HistoryIdError(Exception):
```

Comment:
This is a very nit-picky but I prefer `PreferentialHistoryError` or `PreferenceHistoryNotFound`.